src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -175,10 +175,26 @@ void CallstackThreadBar::OnPick(int x, int y) {
 }
 
 void CallstackThreadBar::SelectCallstacks() {
-  Vec2& from = mouse_pos_last_click_;
-  Vec2& to = mouse_pos_cur_;
+  Vec2 from = mouse_pos_last_click_;
+  Vec2 to = mouse_pos_cur_;
 
-  time_graph_->SelectCallstacks(from[0], to[0], GetThreadId());
+  if (from > to) {
+    std::swap(from, to);
+  }
+
+  uint64_t t0 = time_graph_->GetTickFromWorld(from[0]);
```

Comment:
I assume this will be moved soon outside time_graph, am I right?

Reply:
Yes, this will be removed from timegraph in the next PR.